### PR TITLE
Update ParDo documentation for Python

### PR DESCRIPTION
**Pull request by hadarhg**
R: @aaltay

### CODE REVIEW

File: src/documentation/programming-guide.md [hunk 1]
```diff
@@ -789,7 +789,7 @@ Side inputs are useful if your `ParDo` needs to inject additional data when proc
 %}
 
 # We can also pass side inputs to a ParDo transform, which will get passed to its process method.
-# The only change is that the first arguments are self and a context, rather than the PCollection element itself.
+# The only change is that the first arguments are self and the PCollection element itself.
```
Comment (aaltay) [inline src/documentation/programming-guide.md]: I do not full understand this sentence. What is the `only change` referring too?

**Comment by aaltay** [comment]
R: @sb2nov

File: src/documentation/programming-guide.md (hunk 1 above)
Comment (sb2nov) [inline src/documentation/programming-guide.md]: Yeah no idea. How about changing the sentence to "The first two arguments for the process method would be self and element."

**Comment by sb2nov** [comment]
LGTM thanks.

**Pull request closed without merging**